custom_components/tuya_local/devices/akai_dryer.yaml

```diff
@@ -0,0 +1,118 @@
+name: Akai Heat Pump Dryer
```

Comment:
Make this name generic (the branded name can stay under products).  This way if another brand matches the same dps layout, it can just reuse this config, without confusingly referencing the Akai brand in the default device name.